src/main/java/cat/udl/eps/softarch/linkapp/repository/MeetRepository.java

```diff
@@ -18,4 +19,7 @@
    */
 
   List<Meet> findByTitleContaining(@Param("title") String title);
+  List<Meet> findByStatus(@Param("status") Boolean status);
+  List<Meet> findByStatusAndFinalMeetDateLessThan(@Param("status") Boolean status, @Param("finalMeetDate") ZonedDateTime meetDate);
```

Comment:
These methods are not tested

Reply:
How should we test these methods? There is no additional logic beyond that defined in the Spring methods.

Reply:
Just use them from the stepdefs, through the API (e.g. `\meets\search\findByStatus`)  or from the MeetRepository instance